### PR TITLE
Change malloc template to TSRalloc (fixes Ubuntu build).

### PR DESCRIPTION
**Pull request by ywkaras**

### CODE REVIEW

**Comment by maskit** [comment]
This change looks fine, but I think the root cause is our malloc in ts.h. Since we removed the namespace, it should have a different name (or how can I use it?).

**Comment by maskit** [comment]
Ah, sorry, I wasn't reading the change carefully. I thought you just used existing TSmalloc instead.

What's the R stand for? Do we have documentation for it?

**Review by cmcfarlen — DISMISSED** [review]

**Review by maskit — DISMISSED** [review]
The documentation need to be updated.
https://github.com/apache/trafficserver/blame/4274bc0328b78945cf2405b0745d5d9355b82bbc/doc/developer-guide/api/functions/TSmalloc.en.rst#L62

**Comment by ywkaras** [comment]
@maskit done, thanks for reminding me about the docs.

**Review by cmcfarlen — APPROVED** [review]

File: include/ts/ts.h [hunk 1]
```diff
@@ -3083,9 +3083,15 @@ TSAction TSNetConnectTransparent(
   struct sockaddr const *to    /**< Address to which to connect. */
 );
 
+/**
+  Allocates contiguous, aligned, raw (no construction) memory for a given number number of instancdes of type T.
```
Comment (randall) [inline include/ts/ts.h:3087]: spelling: instances

**Comment by ywkaras** [comment]
OK typo is fixed.

**Review by randall — APPROVED** [review]

**Pull request merged**